### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by nopparuj917**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,11 +17,13 @@
     "@types/react": "18.0.27",
     "@types/react-dom": "18.0.10",
     "axios": "^1.3.2",
+    "dayjs": "^1.11.7",
     "eslint": "8.32.0",
     "eslint-config-next": "13.1.5",
     "next": "13.1.5",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.7.1",
```
Comment (nnanthicha) [inline package.json:26]: use icon from heroicons instead

**Review by nnanthicha — CHANGES_REQUESTED** [review]

**Comment by nopparuj917** [comment]
@nnanthicha doooone

File: src/pages/user/home.tsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
+import Calendar from "@/components/calendar";
 import { PlusIcon } from "@heroicons/react/24/outline";
 import Link from "next/link";
 
 const Home = () => {
   return (
-    <main className="h-screen bg-backgroundColor">
-      <div className="w-full h-2/3 p-5 flex items-center justify-center"></div>
+    <main className="h-full w-full bg-backgroundColor">
```
Comment (nnanthicha) [inline src/pages/user/home.tsx:7]: Homepage background color in preview is not cover full page. Please fix

**Review by nnanthicha — CHANGES_REQUESTED** [review]

**Comment by nopparuj917** [comment]
@nnanthicha @nnanthicha @nnanthicha

**Review by nnanthicha — APPROVED** [review]

**Pull request merged**